pkg/api/decode_aws.go

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (C) 2022 IBM, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package api
+
+type EncodeAwsStruct struct {
```

Comment:
can you add the struct to https://github.com/netobserv/flowlogs2metrics/blob/main/pkg/api/api.go so it will be added to the documentation when running `make generate-configuration`

Reply:
done